tests/bluetooth/shell/audio.conf

```diff
@@ -101,3 +101,4 @@ CONFIG_BT_DEBUG_MEDIA_PROXY=y
 CONFIG_BT_DEBUG_ASCS=y
 CONFIG_BT_DEBUG_PACS=y
 CONFIG_BT_AUDIO_DEBUG_UNICAST_SERVER=y
+CONFIG_TEST_RANDOM_GENERATOR=y
```

Comment:
Why is this needed? (Same comment applies to `prj.conf` in same directory.

Reply:
Without this I get a linker error for `z_impl_sys_rand32_get`.

CI: https://github.com/zephyrproject-rtos/zephyr/runs/5037674015?check_suite_focus=true

Reply:
If there is an issue here, that should be fixed in a separate PR. 

We'll need to look into this I guess.

Reply:
The issue is introduced in this PR because I added  a call to `sys_rand32_get()` in `att.c`.

Reply:
> The issue is introduced in this PR because I added a call to `sys_rand32_get()` in `att.c`.

If this adds a dependency, then that should be added to e.g. CONFIG_BT_EATT. 

The `CONFIG_TEST_RANDOM_GENERATOR` is only for testing, and shouldn't really be used in this context.

Reply:
It seemed it was only the shell application that failed for native_posix.

Reply:
Then this should be moved to the `tests/bluetooth/shell/boards/native_posix.conf` file. 

But I see that this is something missing. The `native_posix.conf` should have `CONFIG_BT_TINYCRYPT_ECC=y`

Reply:
Adding `CONFIG_BT_TINYCRYPT_ECC=y` to `native_posix,conf` does not fix the issue. Moving `CONFIG_TEST_RANDOM_GENERATOR` to the board file seems to work. Also need to create a `native_posix_64.conf` file.

Will try a CI run.

Reply:
CI failed for `native_posix` anyway. Also a couple failures for `qemu_cortex_m3`.

Reply:
Digging around Kconfig I think I have a solution. `BT_EATT` must select `ENTROPY_GENERATOR`.

Reply:
Seems to work, CI passed.